client/src/index.jsx

```diff
@@ -1,15 +1,39 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import axios from 'axios';
 
 class App extends React.Component {
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      images: []
+    }
+    this.getProductImages = this.getProductImages.bind(this);
+  }
+
+  getProductImages() {
+    
+    axios.get(`/photos?product_id=${1}`)
+      .then(response => this.setState({ images: response.data }))
+      .catch(err => console.log(err))
+  }
+
+  componentDidMount() {
+    this.getProductImages();
   }
   
   render() {
+    const { images } = this.state;
+    let display = images.map(image => {
+      console.log(image)
+      return <img src={`https://${image}`}></img>
```

Comment:
I believe you need to add keys to each of the images when you map through  your array!